pdq/flow_tools/basic_flow.py

```diff
@@ -18,9 +19,35 @@ class BasicFlowOpts:
     explore: bool = False
     inline: bool = False
     adk_name: str = 'freepdk-45nm'
+    macros: str = ""
+
+
+# Takes in a path and gets all macro files present there
+def get_macro_files(path):
+    macro_file_list = []
+    # If it's a file, copy it and add it to the filename list
+    if os.path.isfile(path):
+        if path.lower().endswith(('.db', '.lef', '.v')):
+            #FileCopy(path, builder.get_relative("macros/outputs/"))
+            macro_file_list.append(path)
+    # If it's a directory, recursively search for macro files
+    elif os.path.isdir(path):
+        for f in os.listdir(path):
+            macro_file_list += get_macro_files(os.path.join(path, f))
+
+    return macro_file_list
+    
```

Comment:
```suggestion
def _get_macro_files(path):
    """Takes in a path and gets all macro files present there."""
    macro_file_list = []
    # If it's a file, copy it and add it to the filename list.
    if os.path.isfile(path):
        if path.lower().endswith(('.db', '.lef', '.v')):
            macro_file_list.append(path)
    # If it's a directory, recursively search for macro files.
    elif os.path.isdir(path):
        for f in os.listdir(path):
            macro_file_list += get_macro_files(os.path.join(path, f))

    return macro_file_list
```

Reply:
Ah I see I left an old commented out line there. But what's the reason for putting a comment in triple quotes?

Reply:
Oh it's a docstring... got it.